File: test_frontend_proxy.js
// Test script to verify frontend proxy is working
const axios = require('axios');

// Test making a request through the proxy
async function testProxy() {
  try {
    console.log('Testing proxy connection...');
    
    // This request will go through the proxy to /api/login/
    const response = await axios.post('http://localhost:3000/api/login/', {
      username: 'Feelin',
      password: 'newpassword'
    });
    
    console.log('Proxy test successful!');
    console.log('Status:', response.status);
    console.log('Data:', response.data);
  } catch (error) {
    console.error('Proxy test failed:');
    console.error('Error:', error.message);
    if (error.response) {
      console.error('Response status:', error.response.status);
      console.error('Response data:', error.response.data);
    }
  }
}

testProxy();

File: test_frontend_proxy.py
import requests
import json

# Test the frontend proxy by making a request to the frontend server
# which should proxy it to the backend
url = "http://localhost:3000/api/login/"
data = {
    "username": "superadmin",
    "password": "SuperAdmin123!"
}

headers = {
    "Content-Type": "application/json"
}

try:
    response = requests.post(url, data=json.dumps(data), headers=headers)
    print(f"Status Code: {response.status_code}")
    print(f"Response: {response.text}")
except Exception as e:
    print(f"Error: {e}")